src/zappy_server_src/core/commands/fork_command.c

```diff
@@ -88,7 +87,7 @@ static void remove_egg(server_data_t *serv, egg_t *egg, entity_t *egg_e)
     dprintf(2, "A new slot (%i) have been open for the team %s.\n",
     team->max_members, egg->team_name);
     entity_wrapper_remove_entity(serv->entities, egg_e);
-    send_entities_list_info(serv);
+    send_map_info(serv);
```

Comment:
Send map info ?

Reply:
because `send_entities_list_info` sends the whole entity_wrapper, here, i only want to send the modified entity list

Reply:
So why the function is called send_map_info() ? And not add_entites_list ?

Reply:
legacy code